src/app/nutrition/dialogs/food-log-rows.tsx

```diff
@@ -245,3 +264,95 @@ export function TotalsRow({
 
   return <MealTypeRows summary={summary} />;
 }
+
+export function NutritionGoalsRow({ totals }: { totals: FoodLogSummary }) {
+  const calories = useAtomValue(caloriesGoalAtom);
+  const carbs = useAtomValue(carbsGoalAtom);
+  const fat = useAtomValue(fatGoalAtom);
+  const fiber = useAtomValue(fiberGoalAtom);
+  const protein = useAtomValue(proteinGoalAtom);
+  const sodium = useAtomValue(sodiumGoalAtom);
+  const useNetCars = useAtomValue(enableNetCarbsAtom);
+  const caloriesPercentage = parseFloat((totals.calories * 100/ calories).toFixed(2));
+  const totalsCarbs = useNetCars ? totals.carbs - totals.fiber : totals.carbs;
+  const summary: MealTypeSummary = {
+    id: MealType.Anytime,
+    name: "Remaining",
+    foods: [
+      {
+        logDate: "",
+        logId: -1,
+        loggedFood: {
+          mealTypeId: MealType.Anytime,
+          amount: totals.calories,
+          foodId:-1,
+          accessLevel: "PRIVATE",
+          calories:totals.calories,
+          unit: { id: 1, name: "kCal", plural: "kCal" },
```

Comment:
Use "Cal" instead of "kCal" (kcal) since that's what we're using elsewhere right now. Possibly we could add an option to display Calories as Cal, kcal, or kJ but let's keep that separate.